guava/src/com/google/common/base/Joiner.java

```diff
@@ -121,14 +121,14 @@ private Joiner(Joiner prototype) {
    * separator between each, to {@code appendable}.
    */
   @CanIgnoreReturnValue
-  public final <A extends Appendable> A appendTo(A appendable, @org.checkerframework.checker.nullness.qual.Nullable Object[] parts) throws IOException {
+  public final <A extends Appendable> A appendTo(A appendable, @Nullable Object[] parts) throws IOException {
     return appendTo(appendable, Arrays.asList(parts));
   }
 
   /** Appends to {@code appendable} the string representation of each of the remaining arguments. */
   @CanIgnoreReturnValue
   public final <A extends Appendable> A appendTo(
-      A appendable, @NullableDecl Object first, @NullableDecl Object second, @org.checkerframework.checker.nullness.qual.Nullable Object... rest)
+      A appendable, @Nullable Object first, @Nullable Object second, @Nullable Object @Nullable ... rest)
```

Comment:
Your changes introduce `@Nullable` before `...`.
I don't see that annotation in either of the upstream versions (in google or in typetools).
But, maybe I am missing it.  Can you tell me why the final `@Nullable` on this line is present?

Reply:
I introduced these changes for I felt that it is possible to receive an array of Nullable objects. But yes, I should remove these changes for now, for it confusing, and is compromising with the readability of merge